budgetkey_data_pipelines/tests/tenders/test_fixtures.py

```diff
@@ -0,0 +1,9 @@
+from ..common import BasePipelineTestcase
+import os
+
+
+class TendersFixturesTestCase(BasePipelineTestcase):
+
+    def test_fixtures(self):
+        self._run_fixture_tests(os.path.join(os.path.dirname(__file__), "fixtures"),
```

Comment:
can we change this so that you add a test method for each of the tests (so that when a test fails, you see a clear indication of which of the tests actually failed)